### PR TITLE
Legend and wording changes

### PR DESCRIPTION
**Pull request by ryanctan**
- Dots description, both for general and for colorBy
- "Dots" changed to "Travelers"
- Color changes
- Legend alignment

### CODE REVIEW

File: public/visualization.component.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Dots feature displays entries
+ * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Travelers feature displays entries
```
Comment (epicfaace) [inline public/visualization.component.ts:2]: @ryanctan I don't know if we need names here (we already have a commit history), but if you're going to put names, you should also add Ryan Tan

**Comment by ceserani** [comment]
Agreed with Ryan—no names needed here;
Or add his, we might still need to change the language in any case


Sent from my iPhone

On Sep 10, 2019, at 9:03 PM, Ashwin Ramaswami <notifications@github.com<mailto:notifications@github.com>> wrote:


@epicfaace commented on this pull request.

________________________________

In public/visualization.component.ts<https://github.com/cestastanford/grandtour/pull/193#discussion_r323051056>:

> @@ -1,5 +1,5 @@
 /*
- * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Dots feature displays entries
+ * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Travelers feature displays entries


@ryanctan<https://github.com/ryanctan> I don't know if we need names here (we already have a commit history), but if you're going to put names, you should also add Ryan Tan

**Comment by ceserani** [comment]
I see why Ryan thought of changing Dots to Travelers but that is not quite right here either--I'll keep thinking 
The changes all look good great and just what I envisioned, but for three things:
--colors might need further refinement: the ones I suggested work I think for gender but not as well for origin 
--the tooltip box for question marks needs some work: on me is the wording of course that i should revise by next week for all Explorer, but the boxes look a little empty and not optimal in space organization as they are now, they need a little work
--I like the wording, the placement and the font of the descriptive sentence in the grey area that tells you what you are looking at, but I put the '...' there to be expanded when clicked to the full description: can we make that work? how best?

**Comment by ceserani** [comment]
we have that issue again: i don't seem to be able to click on the question mark, once it's below the bubble of a dot:
image

**Comment by ceserani** [comment]
wrogn image above i was talking about this:
Screen Shot 2019-09-10 at 9 51 30 PM

**Pull request merged**